graphql-schemas/schema.yml

```diff
@@ -3217,6 +3218,30 @@ confs:
   - { name: origin, type: string, isRequired: true }
   - { name: records, type: DnsRecord_v1, isList: true }
 
+- name: CloudflareDnsRecord_v1
+  datafile: /cloudflare/dns-record-1.yml
+  fields:
+  - { name: schema, type: string, isRequired: true }
+  - { name: path, type: string, isRequired: true }
+  - { name: name, type: string, isRequired: true }
+  - { name: type, type: string, isRequired: true}
+  - { name: ttl, type: int , isRequired: true}
+  - { name: value, type: string , isRequired: true}
+  - { name: proxied, type: boolean }
+
+
+- name: CloudflareDnsZone_v1
+  datafile: /cloudflare/dns-zone-1.yml
+  fields:
+  - { name: schema, type: string, isRequired: true }
+  - { name: path, type: string, isRequired: true }
+  - { name: zone, type: string, isRequired: true }
+  - { name: partial, type: boolean }
+  - { name: jump_start, type: boolean }
```

Comment:
I'm not sure we have a real need or use case for this feature. I would remove it and leave it as default (it's optional in terraform)